src/docs.md

```diff
@@ -10,12 +10,17 @@ Cadenza JS is a JavaScript library to use the [disy Cadenza](https://www.disy.ne
 
 Cadenza JS is included in the Cadenza distribution in the corresponding version.
 
-Alternatively you can install the most recent version using npm:
+Alternatively you can install the most recent version for a particular Cadenza Release using npm:
 
 ```bash
-npm install @disy/cadenza.js
+npm install @disy/cadenza.js@~10.2.0 # For latest version for Cadenza 10.2 
 ```
 
+The Cadenza Version is reflected in the cadenza.js version (e.g. 10.2.0 for Cadenza 10.2), while the last version segment is increased for both, bugfixes and functional changes.
+
+### Cadenza 10.1 and earlier
```

Comment:
I tend to leave this out, because it will be of interest only for a short time, I guess. We also have this info in the changelog.

Reply:
I'd rather keep it, since we just released 10.1 and may need to bugfix / extend it for a while.
(One other change is still pending for 10.1).

Last decision?